test.c

```diff
@@ -2243,7 +2243,7 @@ upgrade_message_fix(char *body, const size_t nread, const size_t nmsgs, ...) {
   va_list ap;
   size_t i;
   size_t off = 0;
- 
+
```

Comment:
It's not boring, I just wasn't sure what your project's stance was on trailing whitespace (my editors are usually configured to remove it globally). Anyway, now I know, I'll make sure I'll keep trailing ws intact from now on :)